### ISSUE TITLE
Change Tone Not Working

### ISSUE THREAD

**Issue by sk044** [RocketChat/Rocket.Chat#20419]
### Description:
Change Tone of Emojis not working

### Steps to reproduce:

1. Go to any channel
2. Click on 'Emojis'
3. Select change tone

### Expected behavior:
Tone should have been changed

### Actual behavior:
Even after changing tone , actual tone not changed

### Server Setup Information:

- Version of Rocket.Chat Server: 3.11.0-develop
- Operating System: Ubuntu
- Deployment Method: 
- Number of Running Instances: 
- DB Replicaset Oplog: 
- NodeJS Version: 12.18.1
- MongoDB Version:4.2.8

### Client Setup Information

- Desktop App or Browser Version:Chrome
- Operating System:Ubuntu

**Comment by yash-rajpal** [comment]
Can you share some image or video of which emojis are not working?

There are some emojis that support multiple tones, and some emojis have only default tone like here
![image](https://user-images.githubusercontent.com/58601732/105876480-0a6e7680-6025-11eb-8e6d-0323a6341246.png)

**Comment by sk044** [comment]
Yes sure , i got it , actually its not buggy , just looks at 1st sight .
But it can be confusing 
![WhatsApp Image 2021-01-26 at 10 53 57 PM](https://user-images.githubusercontent.com/54908788/105880643-9b475100-6029-11eb-82f5-5d6083fbdeba.jpeg)

We can use someting better like a hover like function. That will be more user friendly.

**Comment by KevLehman** [comment]
This pull request has been inactive for a while, so we’re closing it to keep the backlog manageable.
If you’d like to continue working on it, feel free to reopen or submit a new PR. Thanks for the contribution!
